closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -2169,13 +2169,20 @@ JNIEXPORT jint JNICALL Java_jdk_crypto_jniprovider_NativeCrypto_ChaCha20Init
         return -1;
     }
 
+    /* first initialization */
```

Comment:
Please explain this comment; perhaps it's now redundant given the comments on lines 2174 and 2180?

Reply:
Yes, this comment is a redundant one. Removed it.
`perhaps it's now redundant given the comments on lines 2174 and 2180?`